src/main/kotlin/link/infra/packwiz/installer/ui/gui/ExceptionListWindow.kt

```diff
@@ -24,6 +24,18 @@ class ExceptionListWindow(eList: List<ExceptionDetails>, future: CompletableFutu
 		fun getExceptionAt(index: Int) = details[index].exception
 	}
 
+	private fun openUrl(url: String) {
+		try {
+			if (Desktop.isDesktopSupported() && Desktop.getDesktop().isSupported(Desktop.Action.BROWSE)) {
+				Desktop.getDesktop().browse(URI(url))
+			} else {
+				Runtime.getRuntime().exec(arrayOf("xdg-open", url));
+			}
+		} catch (e: IOException) {
+			// lol the button just won't work i guess
+		} catch (e: URISyntaxException) {}
+	}
```

Comment:
I run KDE so browse doesn't work for me: https://bugs.launchpad.net/ubuntu/+source/openjdk-8/+bug/1574879